### DISCUSSION TITLE
[Help]: Multiple warnings that log_device_status_push not having a handler

### DISCUSSION THREAD

**Discussion by puterfixer** [mbentley/docker-omada-controller#413] [Help]
### Controller Version

5.13.30.8

### Describe Your Issue or Question

Hello,

I'm running a SFF PC with an i5 7th gen, Rocky Linux 9.3 64-bit, docker and the latest omada controller connected to the Host network (no port mapping and NAT needed in docker's SDN).

All is well and functional, it's just a question that nags me: why do I see in the container's log a warning as follows, every 5 minutes? See logs below.

Many thanks in advance.

### Expected Behavior

No warnings in the log unless there's something I need to do about them :)

### Steps to Reproduce

Run container, check container logs.

### How You're Launching the Container

```plain
docker run -d --name omada-controller --restart unless-stopped --ulimit nofile=4096:8192 --net host -e MANAGE_HTTP_PORT=8088 -e MANAGE_HTTPS_PORT=8043 -e PGID="508" -e PORTAL_HTTP_PORT=8088 -e PORTAL_HTTPS_PORT=8843 -e PORT_ADOPT_V1=29812 -e PORT_APP_DISCOVERY=27001 -e PORT_DISCOVERY=29810 -e PORT_MANAGER_V1=29811 -e PORT_MANAGER_V2=29814 -e PORT_TRANSFER_V2=29815 -e PORT_RTTY=29816 -e PORT_UPGRADE_V1=29813 -e PUID="508" -e TZ=Europe/Berlin -v omada_data:/opt/tplink/EAPController/data -v omada_logs:/opt/tplink/EAPController/logs mbentley/omada-controller:latest
```

### Container Logs

```plain
03-01-2024 11:06:15.157 WARN [quartzScheduler_Worker-2] [] c.t.s.c.s.c.TaskExecutorService(): receive scheduled event with identity (log_device_status_push, null) but did not execute because corresponding handler log_device_status_push doesn't exist
03-01-2024 11:11:15.157 WARN [quartzScheduler_Worker-2] [] c.t.s.c.s.c.TaskExecutorService(): receive scheduled event with identity (log_device_status_push, null) but did not execute because corresponding handler log_device_status_push doesn't exist
03-01-2024 11:16:15.157 WARN [quartzScheduler_Worker-1] [] c.t.s.c.s.c.TaskExecutorService(): receive scheduled event with identity (log_device_status_push, null) but did not execute because corresponding handler log_device_status_push doesn't exist
03-01-2024 11:21:15.157 WARN [quartzScheduler_Worker-1] [] c.t.s.c.s.c.TaskExecutorService(): receive scheduled event with identity (log_device_status_push, null) but did not execute because corresponding handler log_device_status_push doesn't exist
03-01-2024 11:26:15.160 WARN [quartzScheduler_Worker-2] [] c.t.s.c.s.c.TaskExecutorService(): receive scheduled event with identity (log_device_status_push, null) but did not execute because corresponding handler log_device_status_push doesn't exist
03-01-2024 11:31:15.157 WARN [quartzScheduler_Worker-2] [] c.t.s.c.s.c.TaskExecutorService(): receive scheduled event with identity (log_device_status_push, null) but did not execute because corresponding handler log_device_status_push doesn't exist
03-01-2024 11:36:15.157 WARN [quartzScheduler_Worker-2] [] c.t.s.c.s.c.TaskExecutorService(): receive scheduled event with identity (log_device_status_push, null) but did not execute because corresponding handler log_device_status_push doesn't exist
03-01-2024 11:41:15.157 WARN [quartzScheduler_Worker-2] [] c.t.s.c.s.c.TaskExecutorService(): receive scheduled event with identity (log_device_status_push, null) but did not execute because corresponding handler log_device_status_push doesn't exist
```

**Comment by mbentley** [comment]
This is likely a bug in the software itself.  I see the same messages logged in my controller.  I don't see any [TP-Link forum](https://community.tp-link.com/en/business/forum/582) posts on the subject yet ([saved search](https://community.tp-link.com/en/business/forum/582?keyword=log_device_status_push)).  Might just be something that should be a debug log.  I can cross post something over there on your behalf and will share the post.

**Reply by mbentley** [reply]
https://community.tp-link.com/en/business/forum/topic/657874?page=1

**Reply by puterfixer** [reply]
You're too kind. Thank you.

I'm glad to hear that it's not just my instance doing this. Since there's no loss of functionality, it wouldn't be an immediate problem. But for someone running this on a RPi with microSD or another SBC with eMMC, writing an unnecessary log line every 5 minutes will gradually kill the storage, unless they do some advanced stuff with ramdisks (where memory is abundent) or other creative solutions. Few people do this.
